etc/eth-contracts/res/EvmErc20V2.hex

```diff
@@ -0,0 +1 @@
15610edd5760405162461bcd60e51b815260206004820152602260248201527f45524332303a206275726e20616d6f756e7420657863656564732062616c616e60448201527f636500000000000000000000000000000000000000000000000000000000000060648201526084016105bd565b6001600160a01b0383166000908152602081905260408120838303905560028054849290610f0c90849061134f565b90915550506040518281526000906001600160a01b038516907fddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef9060200160405180910390a36106ff565b828054610f6390611392565b90600052602060002090601f016020900481019282610f855760008555610fcb565b82601f10610f9e57805160ff1916838001178555610fcb565b82800160010185558215610fcb579182015b82811115610fcb578251825591602001919060010190610fb0565b50610fd7929150610fdb565b5090565b5b80821115610fd75760008155600101610fdc565b600082601f830112611000578081fd5b813567ffffffffffffffff8082111561101b5761101b6113e3565b604051601f8301601f19908116603f01168101908282118183101715611043576110436113e3565b8160405283815286602085880101111561105b578485fd5b8360208701602083013792830160200193909352509392505050565b600060208284031215611088578081fd5b8135611093816113f9565b9392505050565b600080604083850312156110ac578081fd5b82356110b7816113f9565b946020939093013593505050565b600080604083850312156110d7578182fd5b82356110e2816113f9565b915060208301356110f2816113f9565b809150509250929050565b600080600060608486031215611111578081fd5b833561111c816113f9565b9250602084013561112c816113f9565b929592945050506040919091013590565b6000806040838503121561114f578182fd5b823561115a816113f9565b9150602083013567ffffffffffffffff811115611175578182fd5b61118185828601610ff0565b9150509250929050565b600080604083850312156110ac578182fd5b600080604083850312156111af578182fd5b823567ffffffffffffffff8111156111c5578283fd5b6111d185828601610ff0565b95602094909401359450505050565b6000806000606084860312156111f4578283fd5b833567ffffffffffffffff8082111561120b578485fd5b61121787838801610ff0565b9450602086013591508082111561122c578384fd5b5061123986828701610ff0565b925050604084013560ff8116811461124f578182fd5b809150509250925092565b60006020828403121561126b578081fd5b5035919050565b60008060408385031215611284578182fd5b50508035926020909101359150565b600081518084526112ab816020860160208601611366565b601f01601f19169290920160200192915050565b600082516112d1818460208701611366565b9190910192915050565b6000600160f81b82526bffffffffffffffffffffffff198560601b1660018301528360158301528251611315816035850160208701611366565b91909101603501949350505050565b6000602082526110936020830184611293565b6000821982111561134a5761134a6113cd565b500190565b600082821015611361576113616113cd565b500390565b60005b83811015611381578181015183820152602001611369565b83811115610cec5750506000910152565b600181811c908216806113a657607f821691505b602082108114156113c757634e487b7160e01b600052602260045260246000fd5b50919050565b634e487b7160e01b600052601160045260246000fd5b634e487b7160e01b600052604160045260246000fd5b6001600160a01b038116811461140e57600080fd5b5056fea26469706673582212209fb2164fbe4cd313d0df109ec605706fc70da77ce5ebc7b3cfb80b30848bfa8b64736f6c63430008030033
```

Comment:
Do we need to keep in git V2?

Reply:
V2 is not currently used, but presumably someday it will be when we eventually upgrade the ERC-20 tokens. It didn't seem harmful to commit them to the repo early.